docs/UserGuide.md

```diff
@@ -255,13 +255,11 @@ Clears all book and user entries from BookFace.
 
 Format: `clear all`
 
-### Changing the theme: `theme` *COMING SOON*
```

Comment:
I think it is fine to the `theme` CLI command as coming soon in v1.5

Reply:
We will probably need to do this for other planned features/feature flaws

Reply:
I see. Thanks for merging master to the branch!